src/main/scala/com/fulcrumgenomics/util/Metric.scala

```diff
@@ -225,11 +228,25 @@ trait Metric extends Product with Iterable[(String,String)] {
     }
   }
 
-  /** Gets an iterator over the fileds of this metric in the order they were defined.  Returns tuples of names and values */
+  /** Gets an iterator over the fields of this metric in the order they were defined.  Returns tuples of names and values */
   override def iterator: Iterator[(String,String)] = this.names.zip(this.values).toIterator
 
+  /** @deprecated use [[formatValue]] instead. */
+  @deprecated("Use formatValue instead.", since="0.5.0")
+  protected def formatValues(value: Any): String = formatValue(value)
+
   /** Override this method to customize how values are formatted. */
-  protected def formatValues(value: Any): String = formatter.format(value)
+  protected def formatValue(value: Any): String = value match {
+    case null      => ""
+    case None      => ""
```

Comment:
`ReflectionUtil.SpecialEmptyOrNoneToken`?

Reply:
I don't think we want that when _formatting_.  That'd result in `:none:` getting written into the text files and I think that'd be confusing.

Reply:
Sorry, right, this is `Metric`.